app/src/main/java/co/develhope/meteoapp/network/wheaterapi/WheaterService.kt

```diff
@@ -0,0 +1,26 @@
+package co.develhope.meteoapp.network.wheaterapi
+
+import co.develhope.meteoapp.network.wheaterapi.dto.DailySummary
+import co.develhope.meteoapp.network.wheaterapi.dto.WeeklySummary
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface WheaterService {
+
+    //endpoint per dailysummary
+    @GET("forecast?hourly=temperature_2m,rain,showers,snowfall,weathercode,windspeed_10m&current_weather=true&timezone=Europe%2FBerlin&start_date=2022-10-03&end_date=2022-10-03")
+    suspend fun getDailyWehaterSummary(
```

Comment:
giusto passarle nell costruttore della funzione , ma poi devi darl iin pasto all'endpoint se no come fa a sapere che date ti servono. tips: dovete sostituire 2022-10-03.
Togliete anche il valore di default non serve in questo caso. E in generale evitateli